indico/htdocs/css/events/lectures/IndicoLecture.css

```diff
@@ -35,12 +35,13 @@ div.lectureEventHeader {
     margin: 0px;
     padding: 20px 30px;
     position: relative;
-    border-radius: 10px 10px 0px 0px;
+
+    border-top-left-radius: 0.2em;
```

Comment:
I just made the radius `2px` in Compass defaults not to depend on font size in my previous PR. It will probably be unnoticeable here, but in case you want to keep it consistent with